tsconfig.json

```diff
@@ -1,14 +1,29 @@
 {
   "compileOnSave": false,
+  "declaration": true,
   "compilerOptions": {
     "downlevelIteration": true,
     "target": "ES6",
     "module": "ES2015",
     "moduleResolution": "node",
-    "allowJs": true,
+    "allowJs": false,
     "importHelpers": true,
+    "allowSyntheticDefaultImports": true,
+    "esModuleInterop": true,
```

Comment:
Why? ESModules should be as modern as possibly, close to es2022

Reply:
I have removed `esModuleInterop`. Only `allowSyntheticDefaultImports` is relevant for jquery. At some point we should remove jquery completely